source/community.applications/usr/local/emhttp/plugins/community.applications/skins/Narrow/skin.html

```
@@ -1,63 +1,68 @@
+    <ul class='nonselectMenu'>
+      <li><?=tr("VERSION");?></li>
+      <li><span id='caInstalledVersion'></span></li>
+      <li class='caMenuItem caChangeLog'><?tr("Change Log");?></li>
+    <?if ($md5Error):?>
+      <li><font size='0'>Modified</font></li>
+    <?endif;?>
+      <li><hr/></li>
+      <li class='caMenuItem debugging'><?tr("Debugging");?></li>
+    </ul>
+  </div>
+  </span>
```

HTMLHint (1.5.0)

[error] Tag must be paired, no start tag: [ </span> ]

(tag-pair)